applications/HDF5Application/custom_io/hdf5_element_data_value_io.cpp

```diff
@@ -112,6 +145,25 @@ class ReadElementVariableFunctor
     }
 };
 
+template <>
+class ReadElementVariableFunctor<Flags>
+{
+public:
+    void operator()(Flags const& rVariable,
+                    std::vector<ElementType*>& rElements,
+                    File& rFile,
+                    std::string const& rPrefix,
+                    std::size_t StartIndex,
+                    std::size_t BlockSize,
+                    std::string const& rVariableName)
+    {
+        Vector<int> data;
+        rFile.ReadDataSet(rPrefix + "/ElementDataValues/" + rVariableName, data,
```

Comment:
```suggestion
        rFile.ReadDataSet(rPrefix + "/ElementalFlags/" + rVariableName, data,
```

I think the name is misleading